### ISSUE TITLE
noindex, nofollow?

### ISSUE THREAD

**Issue by IschaGast** [pa11y/pa11y-dashboard#140]
I have got my dashboard running and it's working pretty good. Only one site that has this meta tag `<meta name="robots" content="noindex, nofollow" />`won't run the pa11y test. Is that correct?

**Comment by rowanmanning** [comment]
Nope, Pa11y should still pick up that URL. Pa11y doesn't respect the `robots` meta as it's not a crawler, and people need to test private pages too.

Does that answer your question? :)

**Comment by IschaGast** [comment]
It answers my question but then I don't know why some sites are not being picked up by pa11y? 

The one that's failing is a https site, maybe that's something?

**Comment by rowanmanning** [comment]
I might be misunderstanding you. With the dashboard, you need to add each individual URL – It won't spider your entire site looking for URLs. Is that what you were expecting?

**Comment by IschaGast** [comment]
That would be nice but that's not what I was expecting.
When I add a url and say start pa11y it doesn't do anything. Waiting, waiting but no results. Other sites give me results.

Is there a tool that does crawl the whole site and gives you great accessibility feedback? No one is going to put all there urls in pa11y I think

**Comment by rowanmanning** [comment]
Oh that's odd. Could you let me know the URL? If you'd rather not post publicly then my [Twitter DMs are open](https://twitter.com/rowanmanning).

**Comment by IschaGast** [comment]
url is: [beta.schiphol.com](https://beta.schiphol.com)

**Comment by 1stevengrant** [comment]
I'm also experiencing the same thing - which I believe is related to https ? BBC fails but http://www.greenviewevangelicalchurch.co.uk/about does not

**Comment by josebolos** [comment]
Hi @IschaGast,

Apologies for the delay. We've been extremely busy lately.

We've been having a look at the issues that you reported with https://beta.schiphol.com and we just released a update to pa11y [pa11y 4.1.0](https://github.com/pa11y/pa11y/blob/master/CHANGELOG.md#410-2016-11-21) that should fix your problem.

The problem seems to be that https://beta.schiphol.com only has `tslv1.1` and `tslv1.2` enabled. Support for tslv1.0 is disabled in the server. As we made tslv1.0 the default in [pa11y/#62](https://github.com/pa11y/pa11y/issues/62) as a workaround for versions of phantom older than 1.9.8, phantomjs (and therefore pa11y) were failing to connect to your site.

In order to fix it, you can install pa11y 4.1.0 or, if this is not possible, you can use the following pa11y config in order to run your tests:

```json
{
	"phantom": {
	    "parameters": {
	        "ssl-protocol": "tlsv1.1"
	    }
	}
}
```

Run the tests as:
`pa11y --config ./config-tlsv1.1.json --standard WCAG2AA https://beta.schiphol.com/en/`
in order to see the results.

Hope that this helps.

**Comment by josebolos** [comment]
@stevieg83 I'm puzzled by your comment that the BBC doesn't work. We've never had any problems running tests against it. Could you please be able to provide the exact URL that's failing for you and we'll check what's going on?

Thanks.
